frontend/src/layout/navigation/navigationLogic.ts

```diff
@@ -75,6 +84,26 @@ export const navigationLogic = kea<navigationLogicType<UserType, SystemStatus>>(
                 return user?.team?.id
             },
         ],
+        demoWarning: [
+            () => [userLogic.selectors.user, organizationLogic.selectors.currentOrganization],
+            (user: UserType, organization: OrganizationType): WarningType => {
+                const yesterday = new Date()
+                yesterday.setDate(new Date().getDate() - 1)
+
+                if (Date.parse(organization.created_at) >= yesterday.getTime() && user.team?.is_demo) {
```

Comment:
We've been using `moment` elsewhere in the codebase - might make sense to use that here as well.